api/HTMLScriptElement.json

```diff
@@ -330,6 +330,53 @@
           }
         }
       },
+      "integrity": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "17"
```

Comment:
The content attribute entry has `partial_implementation` for Edge. Can you check why that is and whether we should have the same here? It should almost certainly only be for Edge 17-18 though.

Reply:
It looks like `partial_implementation` is marked because some of the [W3C tests fail](https://w3c-test.org/subresource-integrity/subresource-integrity.html), and was added in #2765.  It should have been added with a note, though...